uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -839,6 +839,11 @@ INPUTS:
 sub insert_into_mri_violations_log {
     my ($this, $valid_fields, $severity, $pname, $candID, $visit_label, $file) = @_;
 
+    # determine the future relative path when the file will be moved to
+    # data_dir/trashbin at the end of the script's execution
+    my $file_path     = $file->getFileDatum('File');
+    my $file_rel_path = NeuroDB::MRI::get_trashbin_file_rel_path($file_path);
```

Comment:
if the severity is a warning, need to update the file path to the assembly directory instead of the trashbin. Figure out how to do that given that the file may not be moved already in the assembly directory.

Reply:
done after insertion of the scan into the files table